fr/coursier/livraison.md

```diff
@@ -0,0 +1,24 @@
+---
+lang: fr
+role: coursier
+title: Livraison
+ref: coursier-home
```

Comment:
`ref` should be the same for English & French versions (i.e the same as in file `en/courier/delivery.md`)